packages/wallet/api/src/sharingUnserializer.js

```diff
@@ -0,0 +1,72 @@
+// @ts-check
+import { Far, makeMarshal } from '@endo/marshal';
+
+/**
+ * Make a pair of unserializers that share low-privilege values.
```

Comment:
please document what is functionally different between the two marshallers returned and how the argument conditions that.

Reply:
Added some documentation